monster-cards/src/index.html

```diff
@@ -4,9 +4,6 @@
 		<meta charset="utf-8">
 		<title>dojo2 Monster Cards</title>
 	</head>
-	<body is="app-projector" data-css-transitions>
-		<header is="app-widget" data-uid="navbar"></header>
-		<main is="app-widget" data-uid="container"></main>
-	</body>
+	<my-app></my-app>
```

Comment:
if not just placing at `document.body` a better name would likely be useful, ie `<monster-app>`